src/airflow/message-broker-setup.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
-kafka/bin/kafka-topics.sh --create --topic app-insights-measurements-topic --partitions 1 --bootstrap-server message-broker:9092
-kafka/bin/kafka-topics.sh --create --topic hourly-measurements-topic --partitions 1 --bootstrap-server message-broker:9092
+kafka/bin/kafka-topics.sh --create --topic "$INSIGHTS_MEASUREMENTS_TOPIC" --partitions 1 --bootstrap-server message-broker:9092
+kafka/bin/kafka-topics.sh --create --topic "$HOURLY_MEASUREMENTS_TOPIC" --partitions 1 --bootstrap-server message-broker:9092
```

Comment:
using envionment variables for topic names

Reply:
cool